ubuntuStation/models.py

```diff
@@ -77,7 +77,8 @@ class DeployedURL(db.Model):
     __tablename__="deployed_url"
     deployed_url_id=db.Column(db.Integer, primary_key=True, autoincrement=True)
     url_text=db.Column(db.String(255))
-    is_kioski=db.Column(db.String(45))
+    is_kioski=db.Column(db.Boolean)
```

Comment:
@ryantanaka is_kioski?

Reply:
This is addressed in an upcoming commit 26d95d4.